frontend/package.json

```diff
@@ -4,11 +4,14 @@
     "ng": "ng",
     "start": "yarn serve",
     "start-docker": "ng serve --host 0.0.0.0 --port 4200 --proxy-config proxy.conf.docker.json",
+    "preserve": "yarn prebuild",
```

Comment:
Why this wrapper around `yarn prebuild`? The command's name can be misleading, I first thought this would be for backing something up [cf. here](https://www.wordnik.com/words/preserve), so if you'd like to retain the pre-build / pre-serve distinction, perhaps introduce a hyphen?

Reply:
Yes, I was also confused by this. Though  the documentation on [pre & post scripts](https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts) does not suggest you could also use `pre-` as a prefix.

Reply:
Ah, didn't realize this was an npm thing.